### PR TITLE
feat: include abi custom error params in the returned error

### PR DESCRIPTION
**Pull request by mrekucci**
### Checklist

- [x] I have read the [coding guide](https://github.com/ethersphere/bee/blob/master/CODING.md).
- [ ] My change requires a documentation update, and I have done it.
- [ ] I have added tests to cover my changes.
- [x] I have filled out the description and linked the related issues.

### Description
Includes abi error params in the returned error.
Example of the output error: `execution reverted: InsufficientBalance(available=0,required=111)` for the following failed transaction: https://goerli.etherscan.io/tx/0x74a2577db1c325c41e38977aa1eb32ab03dfa17cc1fa0649e84f3d8c0f0882ee

### CODE REVIEW

File: pkg/transaction/transaction.go [hunk 1]
```diff
@@ -596,20 +597,53 @@ func (t *transactionService) UnwrapABIError(ctx context.Context, req *TxRequest,
 		return nil
 	}
 
-	res, cErr := t.Call(ctx, req)
-	if cErr != nil {
+	_, cErr := t.Call(ctx, req)
+	var derr rpc.DataError
```
Comment (istae) [inline pkg/transaction/transaction.go:601]: i would move this above line 606

**Review by istae — APPROVED** [review]

File: go.mod [hunk 2]
```diff
@@ -32,7 +32,7 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/prometheus/client_golang v1.14.0
 	github.com/spf13/afero v1.6.0
-	github.com/spf13/cobra v1.0.0
+	github.com/spf13/cobra v1.5.0
```
Comment (janos) [inline go.mod:35]: It looks to me that this upgrade is not needed for this PR.

Also, there are new dependencies added that do not look relevant for the changes in the code.

Comment (mrekucci) [reply go.mod:35]: So I used `go get github.com/ethereum/go-ethereum@v1.13.2` and then `go mod tidy` to update the `go-ethereum` dependency. Everything else was updated automatically by the `go mod` system.

Comment (janos) [reply go.mod:35]: Thanks.

File: pkg/transaction/transaction.go (hunk 1 above)
Comment (mrekucci) [reply pkg/transaction/transaction.go]: Done.

**Pull request merged**